components/QrScanner.tsx

```diff
@@ -88,6 +101,9 @@ export const QrScanner: React.FC<QrScannerProps> = (props) => {
           }}>
           <Image source={Theme.CameraFlipIcon} />
         </TouchableOpacity>
+        <Text align="center" weight="semibold" margin="9 0">
```

Comment:
please extract the styles to theme files.

Reply:
Following from the previous code, I'm passing these 3 as props into <Text> which are already assigned as TextProps in the component **Text.tsx**